### DISCUSSION TITLE
How to handle sub tags?

### DISCUSSION THREAD

**Discussion by wintermeyer** [wintermeyer/animina#1055] [Ideas]
I would like to reorganise the tag system. I would like to e.g. create tags for car brands but those are not of importance for many people. So I'd like to have an opportunity to open a subcategory of tags within the white, green and red flag system which displays those car brands. So if there is a Porsche enthusiast he/she can open those subcategories and click on Porsche. But for someone who couldn't care less about cars that whole subset wouldn't even exist. Ideally we can create multiple levels for this.

But I have no idea how to change the data structure for this. 

@briankariuki @MICHAELMUNAVU83 @Kimutai01 Any ideas?

**Comment by MICHAELMUNAVU83** [comment]
@wintermeyer  , are you open to creating a sperate table to hold these sub tags? This would inform my thinking

**Reply by wintermeyer** [reply]
> @wintermeyer , are you open to creating a sperate table to hold these sub tags? This would inform my thinking

I am open to nearly anything. 😉

But what about our white, green and red SQL queries? Wouldn't an other table make our live more complex here?
